app/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/meldekort/domene/MeldekortBehandling.kt

```diff
@@ -355,3 +361,40 @@ fun Sak.opprettMeldekortBehandling(
         ),
     )
 }
+
+fun Sak.opprettMeldekortKorrigering(
+    saksbehandler: Saksbehandler,
+    forrigeBehandling: MeldekortBehandling,
+): MeldekortBehandling.MeldekortUnderBehandling {
+    require(this.meldekortBehandlinger.contains(forrigeBehandling)) {
+        "Forrige behandling ${forrigeBehandling.id} tilhører ikke denne saken"
+    }
+
+    val meldekortId = MeldekortId.random()
+    val meldeperiode = hentSisteMeldeperiodeForKjede(forrigeBehandling.kjedeId)
+
+    return MeldekortBehandling.MeldekortUnderBehandling(
+        id = meldekortId,
+        sakId = this.id,
+        saksnummer = this.saksnummer,
+        fnr = this.fnr,
+        opprettet = nå(),
+        // TODO: bør navkontor hentes på nytt? Kan den være endret innenfor samme meldeperiode?
+        navkontor = forrigeBehandling.navkontor,
+        ikkeRettTilTiltakspengerTidspunkt = forrigeBehandling.ikkeRettTilTiltakspengerTidspunkt,
+        // TODO: må kunne sette et annet bruker-meldekort når vi skal støtte korrigering fra bruker
+        brukersMeldekort = forrigeBehandling.brukersMeldekort,
+        meldeperiode = meldeperiode,
+        saksbehandler = saksbehandler.navIdent,
+        type = MeldekortBehandlingType.KORRIGERING,
+        // TODO: forhåndsutfylle denne fra forrige meldekortbehandling?
+        beregning = MeldeperiodeBeregning.IkkeUtfyltMeldeperiode.fraPeriode(
+            meldeperiode = meldeperiode,
+            meldekortId = meldekortId,
+            sakId = this.id,
+            // TODO: dette er kanskje også en variabel som saksbehandler kan korrigere? (På sikt?)
+            maksDagerMedTiltakspengerForPeriode = forrigeBehandling.beregning.maksDagerMedTiltakspengerForPeriode,
```

Comment:
Denne kommer fra meldeperioden, men jeg vet ikke helt hva vi bruker den til i beregningen. Den er jo lagret i meldeperioden og vi peker jo på meldeperioden så sånn sett er vi good. Den bør ikke brukes i beregningen siden saksbehandler kan overstyre dette for manuell behandling. Men for automatisk behandling/beregning bør ikke algoritmen kunne overstyre dette!